osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/NewConnectionRequestThrottler.java

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: Copyright Contributors to the GXF project
+//
+// SPDX-License-Identifier: Apache-2.0
+package org.opensmartgridplatform.throttling;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class NewConnectionRequestThrottler {
+  private final Semaphore semaphore;
+  private final int maxNewConnections;
+  private final long resetTimeInMs;
+  private final long maxWaitForNewConnectionsInMs;
+  private final Timer resetNewConnectionRequestsTimer;
```

Comment:
Wordt alleen in de constructor gebruikt.

Reply:
Scherp, aangepast